Assembler/README.md

```diff
@@ -1,5 +1,5 @@
 # Python-Assembler
-# WE NEED A FREE T-SHIRT
+# WE need A FREE T-SHIRT
```

Comment:
I'd say leave it as it was since lower case only one word results in _inconsistency_. And the change made does not make any sense.